src/bin/sol-fbp-runner/web-inspector.js

```diff
@@ -16,451 +16,178 @@
-    this.html.start.disabled = false;
-    this.html.finish.disabled = true;
 };
 
 FBPInspector.prototype.el = function (id) {
     return document.getElementById(id);
 };
 
-FBPInspector.prototype.clearTableRows = function (domElem) {
-    while (domElem.rows.length)
-        domElem.deleteRow(0);
-};
-
-FBPInspector.prototype.addTableRow = function (domElem, idx, rowSpec) {
-    var r = domElem.insertRow(idx);
-    for (var i = 0; i < rowSpec.length; i++) {
-        var c = r.insertCell(i);
-        var rs = rowSpec[i];
-        if (rs.content)
-            c.textContent = rs.content;
-        if (rs.html)
-            c.innerHTML = rs.html;
-        c.className = rs.className;
-    };
-};
 
 FBPInspector.prototype.start = function () {
-    this.nodes = {};
-    this.events = [];
 
     this.source = new EventSource(window.location.origin + '/events');
     this.source.fbpInspector = this;
     this.source.onopen = function (e) {
-        this.fbpInspector.clearTableRows(this.fbpInspector.html.nodes);
-        this.fbpInspector.logDeliveredClear();
-        this.fbpInspector.logSentClear();
-        this.fbpInspector.html.start.disabled = true;
-        this.fbpInspector.html.finish.disabled = false;
+        //TODO
     };
 
     this.source.onerror = function (e) {
         this.fbpInspector.finish();
     };
 
+    var ev;
     this.source.onmessage = function (e) {
-        var ev = JSON.parse(e.data);
-        this.fbpInspector.feedEvent(ev);
-    };
-};
-
-FBPInspector.prototype.logSentClear = function () {
-    this.clearTableRows(this.html.logSent);
-};
-
-FBPInspector.prototype.logDeliveredClear = function () {
-    this.clearTableRows(this.html.logDelivered);
-};
-
-FBPInspector.prototype.getNodeName = function (node) {
-    if (this.nodes[node])
-        return this.nodes[node].creation.payload.id || node;
-    return node;
-};
-
-FBPInspector.prototype.renderPacket = function (packet) {
-    var packet_type = packet.packet_type;
-    var payload = packet.payload;
-    var extraClass = "";
-    var body = "";
-
-    if (packet_type == "empty" ||
-        packet_type == "any") {
-    } else if (packet_type == "boolean" ||
-               packet_type == "byte" ||
-               packet_type == "timestamp") {
-        body = payload;
-    } else if (packet_type == "string") {
-        body = payload; // TODO: escape
-    } else if (payload instanceof Array) {
-        extraClass += " fbp-packet-composed";
-        for (var i = 0; i < payload.length; i++) {
-            if (i > 0)
-                body += '<span class="fbp-packet-composed-separator">, </span>';
-            body += this.renderPacket(payload[i]);
-        }
-    } else {
-        var keys = [];
-        for (var key in payload) {
-            if (!payload.hasOwnProperty(key))
-                continue;
-            keys.push(key);
+        try {
+            feed(e.data);
+        } catch (err){
+            var fixed = e.data.toString().replaceAll(":inf,",':"inf",');
```

Comment:
Yes, we got an error when trying to read a port with the "inf" value. The value itself is lacking the double quotes around it, so we are receiving inf instead of "inf" and it is messing with the JSON Parser. I added this try/catch to temporarily solve this issue.